.goreleaser.yml

```diff
@@ -34,6 +34,14 @@ archives:
     amd64: x86_64
 checksum:
   name_template: 'checksums.txt'
+signs:
+  - cmd: cosign
+    env:
+    - COSIGN_EXPERIMENTAL=1
+    certificate: '{{ trimsuffix .Env.artifact ".txt" }}.pem'
+    signature: "${artifact}.sig"
+    args: ["sign-blob", "--oidc-issuer=https://token.actions.githubusercontent.com", "--output-signature=${signature}", "--output-certificate=${certificate}",  "${artifact}"]
```

Comment:
Note: this will not work with cosign v1.3.1, because the implemented feature is not part of that release.
CC: @dlorenc can we get a new cosign release, soon? :D

Reply:
I ran into the same issue with my new kubectl plugin... cosign will fail with "unknown flag --output-signature" with version v1.3.1

Reply:
this is solved with cosign 1.4.1

Reply:
thank you 🙋🏻‍♂️